CMakeLists.txt

```diff
@@ -302,4 +305,9 @@ foreach(plib_dest xmls filters waveforms block_diagrams)
 	install(DIRECTORY ${plib_dest} DESTINATION ${PLIB_DEST})
 endforeach()
 
+if (WITH_PLUGIN_SPECTRUM_ANALYZER)
+	add_definitions(-DPLUGIN_SPECTRUM_ANALYZER)
+endif()
+
+
```

Comment:
I guess this option should be in the plugins specific CMakeLists file. Moreover, I would just not compile the library at all if the option is OFF. As it is the default value, I would remove the plugin from the list of plugins and just add it `if (WITH_PLUGIN_SPECTRUM_ANALYZER)`

Reply:
I did that initially but as Michael said we need to build the plugin. As for the option, I prefer all options to be at top level cmake.

Reply:
Cannot say I agree with that but fair enough...